src/EnergyPlus/PlantLoopEquip.cc

```diff
@@ -312,7 +312,7 @@ namespace PlantLoopEquip {
                 sim_component.compPtr->simulate(sim_component_location, FirstHVACIteration, CurLoad, RunFlag);
 
             } else if (EquipTypeNum == TypeOf_PipingSystemPipeCircuit) {
-                SimPipingSystemCircuit(sim_component.Name, sim_component.CompNum, InitLoopEquip, FirstHVACIteration);
+                SimPipingSystemCircuit(sim_component.Name, sim_component.CompNum, FirstHVACIteration, InitLoopEquip);
```

Comment:
For reference, [here](https://github.com/NREL/EnergyPlus/blob/4bc63b707f3c1ea9291ee41fbf5f8b344998b4a7/src/EnergyPlus/PlantPipingSystemsManager.cc#L163-L166) is the signature of the function:

```
    void SimPipingSystemCircuit(std::string const &EquipName,
                                int &EqNum,
                                bool const EP_UNUSED(FirstHVACIteration),
                                bool const InitLoopEquip)
```

Obviously passing init loop equip first was **wrong**.